Changelogs/1.3.0

```diff
@@ -0,0 +1,13 @@
+1.3.0 Release notes (2023-11-02)
+=============================================================
+
+Improvements to the 'RingPublishingTracking' module.
+
+### Features
+
+* Implemented video event tracking flow
```

Comment:
@danielcalka Tu raczej trzeba wpisać to co jest publicznie dostępna dla klienta modułu, sorry jeśli nie wyraziłem się jasno :)
wpis "Implemented video events factory" nic nie mówi, spojrzyj na changelog iOS

Reply:
Poprawione, dzięki wielkie!